config/default.yml

```diff
@@ -151,6 +151,7 @@ PreCommit:
     flags: ['-IHn', "\t"]
     exclude:
       - '**/*.go'
+      - 'Godeps/Godeps.json'
```

Comment:
Not sure about the preferred alphabetization here. I grouped it with the previous `.go` ignore for logical grouping but maybe we're going for alphabetization here?